src/PowerShellEditorServices/Services/PowerShellContext/EditorOperationsService.cs

```diff
@@ -158,21 +200,41 @@ public string GetWorkspaceRelativePath(string filePath)
 
         public async Task ShowInformationMessageAsync(string message)
         {
+            if (!TestHasLanguageServer())
+            {
+                return;
+            };
+
             await _languageServer.SendRequest<string>("editor/showInformationMessage", message);
         }
 
         public async Task ShowErrorMessageAsync(string message)
         {
+            if (!TestHasLanguageServer())
+            {
+                return;
+            };
+
             await _languageServer.SendRequest<string>("editor/showErrorMessage", message);
         }
 
         public async Task ShowWarningMessageAsync(string message)
         {
+            if (!TestHasLanguageServer())
+            {
+                return;
+            };
+
             await _languageServer.SendRequest<string>("editor/showWarningMessage", message);
         }
 
         public async Task SetStatusBarMessageAsync(string message, int? timeout)
         {
+            if (!TestHasLanguageServer())
+            {
+                return;
+            };
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this empty statement.](https://app.codacy.com/manual/TylerLeonhardt/PowerShellEditorServices/pullRequest?prid=4787682)